pallets/xcm/src/lib.rs

```diff
@@ -147,7 +147,7 @@ impl<T: Config> DmpMessageHandler for LimitAndDropDmpExecution<T> {
 	) -> Weight {
 		let mut used = 0;
 		for (_sent_at, data) in iter {
-			let id = sp_io::hashing::twox_64(&data[..]);
+			let id = sp_io::hashing::blake2_256(&data[..]);
```

Comment:
This hashing algorithm was changed to Blake 2 256 to be in line with how every other XCM function hashes the versioned message. Not sure if it changes anything, but would like to point it out here in case there's a special reason why Twox64 MUST be used.

Reply:
no - blake2 should be fine.